man/docker-info.1.md

```diff
@@ -11,9 +11,9 @@ docker-info - Display system-wide information
 
 # DESCRIPTION
 This command displays system wide information regarding the Docker installation.
-Information displayed includes the number of containers and images, pool name,
-data file, metadata file, data space used, total data space, metadata space used
-, total metadata space, execution driver, and the kernel version.
+Information displayed includes the number of containers and (non-tagged) images,
```

Comment:
@SvenDowideit I agree. We should have it at both places.
In the docs and `docker info` output.